datalad_container/containers_run.py

```diff
@@ -131,14 +133,36 @@ def __call__(cmd, container_name=None, dataset=None,
             # just prepend and pray
             cmd = container['path'] + ' ' + cmd
 
+        extra_inputs = []
+        for extra_input in ensure_iter(container.get("extra-input",[]), set):
+            try:
+                xi_kwargs = dict(
+                    img_dspath=image_dspath,
+                    img_dirpath=op.dirname(image_path) or ".",
+                )
```

Comment:
codeclimate btw warned about code duplication here. I think it is ok, although indeed duplicates with cmd_kwargs definition above so we could have some `common_kwargs` to be this structure and defined before any use and then only enhanced for `cmd_kwargs`.  But really might be not worth the hassle this far, just wanted to leave a note that I am ok to suppress codeclimate for this.